Makefile

```diff
@@ -49,7 +49,7 @@ e2etests: ## Run the e2e suite against your local cluster
 		--ginkgo.focus="${FOCUS}" \
 		--ginkgo.timeout=30m \
 		--ginkgo.grace-period=5m \
-		--ginkgo.vv	
+		--ginkgo.vv || true
```

Comment:
Can we remove this || true? We want to make sure we understand when it fails.

Reply:
If we know of tests that aren't consistently succeeding, we can add a Skip the test until we figure it out.